account_check_printing_report_base/migrations/18.0.1.0.0/pre-migration.py

```diff
@@ -6,14 +6,13 @@
 
 @openupgrade.migrate()
 def migrate(env, version):
-    openupgrade.rename_fields(
-        env,
-        [
-            (
-                "account.journal",
-                "account_journal",
-                "account_check_printing_layout",
-                "bank_check_printing_layout",
-            ),
-        ],
-    )
+    if openupgrade.column_exists(
+        env.cr, "account_journal", "account_check_printing_layout"
+    ):
```

Comment:
```suggestion
```
I don't think this is necessary, because the module already depends on account, so the data will already exist. ping @pedrobaeza

Reply:
Ok.
Remove the code, thanks